src/main/groovy/org/gradle/android/workarounds/room/argumentprovider/RoomSchemaLocationArgumentProvider.groovy

```diff
@@ -11,20 +12,25 @@ import org.gradle.process.CommandLineArgumentProvider
 
 @CompileStatic
 abstract class RoomSchemaLocationArgumentProvider implements CommandLineArgumentProvider {
+
+    @Internal
+    final File projectDir;
```

Comment:
Storing the `projectDir` instead of the `Project` to be compatible with the configuration cache.